library/nonenglish_helpers.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * User: shaharzi
+ * Date: 02/11/16
+ * Time: 10:05
+ */
+
+
+/**
+ * Basename functionality for nonenglish languages (without this, basename function ommits nonenglish characters).
+ * @param $path
+ * @return $string
+ */
+function basename_nonenglish($path){
```

Comment:
ongoing discussion for this issue is here:
https://sourceforge.net/p/openemr/discussion/202506/thread/d338ad7b/